DoSomething/gambit-conversations
# Pass Twilio From Number instead of Messaging Service Sid

#### What's this PR do?

* Passes a `from` parameter instead of `messagingServiceSid` when posting to the Twilio API to deliver a message to a User.

* Adds `test/lib/twilio.test.js`

#### How should this be reviewed?
* Staging: send a message to Gambit and verify response is returned.
* Localhost: 
    * Add the staging number as your `TWILIO_FROM_NUMBER` in `env.`
    * `POST /receive-message` passing your mobile number as the `From`
    * Verify Gambit reply is sent to your mobile via staging number

#### Any background context you want to provide?
Hopes to improve queue times when we're sending a large broadcast (refs https://github.com/DoSomething/blink/pull/171)

#### Relevant tickets
Fixes #251

#### Checklist
- [x] Added staging `TWILIO_FROM_NUMBER`
- [x] Added production `TWILIO_FROM_NUMBER`
